backoffice/core/views.py

```diff
@@ -1,17 +1,22 @@
 from rest_framework import viewsets
 from rest_framework.parsers import FormParser, MultiPartParser
+from rest_framework.decorators import permission_classes
 from django.contrib.auth.hashers import make_password
+from django.contrib.auth.models import Group, Permission
 from core.models import (
```

Comment:
Por qué el import con `(` ?

Reply:
Porque leí esto: "The Python standard library is conservative and requires limiting lines to 79 characters (and docstrings/comments to 72). The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation." [1]

Como la línea del import pasaba los 79 caracteres (y también leí que import * es mala práctica [2]) utilicé paréntesis.

[1] https://www.python.org/dev/peps/pep-0008/#maximum-line-length
[2] https://stackoverflow.com/questions/2386714/why-is-import-bad

Reply:
Ok, excelente